src/test/java/tests/mocks/MockClient.java

```diff
@@ -12,65 +12,65 @@
 /**
  * A MockClient is used for creating an HTTP request to test
  * parts of the httpserver.
- * 
- * It has an interface similar to {@link httpserver.HTTPRequest}
- * 
- * @see httpserver.HTTPRequest
+ *
+ * It has an interface similar to {@link httpserver.HttpRequest}
+ *
+ * @see httpserver.HttpRequest
  */
 public class MockClient {
-  
+
   public static int DESIRED_PORT = 4444;
```

Comment:
Is there some way to make a socket that doesn't use IP?

Basically, instead of having HttpRequest, HttpResponse, and MockClient use an actual ServerSocket, is there something that's basically just a text buffer that can be used instead?